docs/spec/reactors/consensus/consensus-reactor.md

```diff
@@ -0,0 +1,364 @@
+# Consensus Reactor
```

Comment:
This seems similar to the previous file, however, after some git digging, it seems like this has been deleted from `main` at some point (I'd say by mistake). I've added a CometBFT-internal AI to find out what happened.

In the meantime, I don't think you want to keep this as part of your fork: I'd "git rm" it

Reply:
Resolved